packages/conf-mingw-w64-nettle-x86_64/conf-mingw-w64-nettle-x86_64.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+homepage: "https://www.gnutls.org"
+authors: ["Nikos Mavrogiannopoulos" "Simon Josefsson"]
+license: "LGPL-2.1-or-later"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+flags: conf
+available: os = "win32"
+build: ["pkgconf" "--personality=x86_64-w64-mingw32" "nettle"]
+depends: [
+  "conf-pkg-config" {build}
+  "conf-mingw-w64-gcc-x86_64" {build}
+]
+synopsis: "nettle for x86_64 mingw-w64"
```

Comment:
Following https://www.lysator.liu.se/~nisse/nettle/ , we can fix this lint error with

```suggestion
synopsis: "Nettle for x86_64 mingw-w64"
```